assets/app/scripts/directives/logViewer.js

```diff
@@ -82,13 +97,68 @@ angular.module('openshiftConsole')
                 });
               }
             };
-            $win.scroll(onScroll);
 
 
-            var onResize = _.debounce(updateScrollLinks, 50);
+
+            var attachScrollEvents = function() {
+              if(window.innerWidth < BREAKPOINTS.screenSmMin) {
+                $win.on('scroll', onScroll);
+                $cachedScrollableNode.off('scroll', onScroll);
+              } else {
+                $win.off('scroll', onScroll);
+                $cachedScrollableNode.on('scroll', onScroll);
+              }
+            };
+
+
+            // the class .target-logger-node is needed to adjust some
+            // css when the target is not the window.
+            // TODO: resize event breaks the affix, even with this if/else.
+            // however, on first load of either mobile or non this works fine.
+            var affix = function() {
+              if(window.innerWidth < BREAKPOINTS.screenSmMin) {
+                $affixableNode
+                  .removeClass('target-logger-node')
+                  .affix({
+                    target:  window,
+                    offset: {
+                        top:  $scope.followAffixTop || 0, // 390,
+                        bottom: $scope.followAffixBottom || 0 // 90
+                    }
+                  });
+              } else {
+                $affixableNode
+                  .addClass('target-logger-node')
+                  .affix({
+                    target:  $cachedScrollableNode,
+                    offset: {
+                        top: $scope.followAffixTop || 0, // 390,
+                        bottom: $scope.followAffixBottom || 0 // 90
+                    }
+                  });
+              }
+            };
+
+
+            // roll up & debounce the various fns to call on resize
+            var onResize = _.debounce(function() {
+                // toggle off the follow behavior if the user resizes the window
+                onScroll();
+                // and udpate scroll handlers
+                detectScrollableNode();
+                attachScrollEvents();
+                updateScrollLinks();
+                affix();
+            });
```

Comment:
Oop, yeah, good catch. Don't want to miss that.  it defaults to 0 if just wanting to kick it up the stack, but def want a real delay here. fixing.  Was 50, setting to 100. 200 even seemed reasonable when testing.